### ISSUE TITLE
Classification: add labels in confusion_matrix.

### ISSUE THREAD

**Issue by kashevg** [ageron/handson-ml2#10]
In the classification chapter, there is a part with a confusion matrix. It really confuses, because there TP stays for False and TN stays for True class.
Also, in the first book (don't have the second) confusion matrix is showed transponed. TP should be always in the upper left corner.

**Comment by ageron** [comment]
Hi @kashevg,

Thanks for your feedback. I suppose you are referring to figure 3-2?

![mlst_0302](https://user-images.githubusercontent.com/76661/58472840-c228f200-8179-11e9-8952-6be6001cb08e.png)

I'm not sure what you mean by "TP stays for False and TN stays for True class"? In the example, we are building a "5-detector", so "positive" means it's a 5, and "negative" means it's not a 5. So a TP is an image that is correctly labeled as positive. The figure does show 5s correctly labeled as 5s in the lower right cell, and that's the cell labeled as TP, so frankly I don't see the problem.

Regarding the confusion matrix, I'm not sure there is any official standard for the order of the columns and rows. I just did a quick informal search on Google Image for "confusion matrix" and among the first 11 confusion matrices for binary classifiers, 6 have the TP in the top left, and 5 in the lower right. That's almost 50%.

Moreover, Scikit-Learn itself puts the TP in the lower right, as the following code demonstrates:

```python
>>> from sklearn.metrics import confusion_matrix
>>> #               1 TN  2 FP   3 FN      4 TP
>>> confusion_matrix([0,  0, 0,  1, 1, 1,  1, 1, 1, 1], # y_true
...                  [0,  1, 1,  0, 0, 0,  1, 1, 1, 1]) # y_pred
...
array([[1, 2],
       [3, 4]])
```

I hope this helps!

**Comment by kashevg** [comment]
Hi, @ageron 
Ok, I get it. 
As for me, it's so unnatural to have TP in the right lower corner. I know, it's a common approach for confusion matrices to sort classes in ascending order, but for binary classification, IMHO, it's better to have "positive" at the beginning, the same order as in [wiki](https://en.wikipedia.org/wiki/Confusion_matrix)
 Thank you.
